spring-boot/src/test/java/org/springframework/boot/context/config/ConfigFileEnvironmentPostProcessorTests.java

```diff
@@ -435,6 +435,18 @@ public void yamlSetsProfiles() throws Exception {
 	}
 
 	@Test
+	public void profilesAddedToEnvironmentAndViaProperty() throws Exception {
+		EnvironmentTestUtils.addEnvironment(this.environment,
+				"spring.profiles.active:foo");
+		this.environment.addActiveProfile("dev");
+		this.initializer.postProcessEnvironment(this.environment, this.application);
+		assertThat(this.environment.getActiveProfiles(),
+				equalTo(new String[] { "foo", "dev" }));
```

Comment:
Yes, `arrayContaining()` is for ordered checks, `arrayContainingInAnyOrder()` not.
But I see that it has been closed.
Have a good day!